compiler/codegen/OMRAheadOfTimeCompile.hpp

```diff
@@ -94,15 +95,17 @@ class OMR_EXTENSIBLE AheadOfTimeCompile
     * it into an IteratedExternalRelocation.
     */
    static void interceptAOTRelocation(TR::ExternalRelocation *relocation) { }
+   static const size_t SIZEPOINTER = sizeof(uintptr_t);
 
-   private:
+private:
    TR::Compilation *                           _comp;
    TR_LinkHead<TR::IteratedExternalRelocation> _aotRelocationTargets;
    uint32_t                                   _sizeOfAOTRelocations;
    uint8_t                                   *_relocationData;
    uint32_t                                  *_aotRelocationKindToHeaderSizeMap;
+   TR::RelocationRuntime*                       _reloRuntime;
```

Comment:
any reason the spacing isn't lined up? Also, please put the asterisk with the name, since that's how it's parsed by the compiler.

Reply:
Fixed that, thank you for the comment!